### PR TITLE
Feature/new shine feature

### PR DESCRIPTION
**Pull request by pingu1m**
Shine new feature added

### CODE REVIEW

**Comment by pingu1m** [comment]
+1 This feature will fix all of your problems

**Review by pingu1m — COMMENTED** [review]
+1 Good code

**Review by pingu1m — COMMENTED** [review]
+1 Looks good.

File: source-code1/2.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This file was modified by user on development branch
+Added on shiny feature branch
```
Comment (pingu1m) [inline source-code1/2.py:2]: + This line looks great.

Comment (fgmonad) [inline source-code1/2.py]: + Very good

**Pull request merged**